pkg/apis/openebs.io/v1alpha1/cstorpool_cluster.go

```diff
@@ -80,6 +80,11 @@ type CStorPoolClusterSpec struct {
 	// If tolerations at PoolConfig is empty, this is written to
 	// CSPI PoolConfig.
 	Tolerations []v1.Toleration `json:"tolerations"`
```

Comment:
Not relevant to this PR. We are using using `Tolerations` from poolConfig. This can be misleading. Can you verify the same?

Reply:
Looks good.